plugins/GSdx/GS.cpp

```diff
@@ -506,15 +506,13 @@ EXPORT_C_(int) GSopen2(void** dsp, uint32 flags)
 	static bool stored_toggle_state = false;
 	bool toggle_state = !!(flags & 4);
 
+	theApp.ClearTempConfig();
+
 	GSRendererType renderer = s_renderer;
 	// Fresh start up or config file changed
 	if (renderer == GSRendererType::Undefined)
 	{
-		renderer = static_cast<GSRendererType>(theApp.GetConfigI("Renderer"));
-#ifdef _WIN32
-		if (renderer == GSRendererType::Default)
-			renderer = GSUtil::GetBestRenderer();
-#endif
+		renderer = theApp.GetTempConfig<GSRendererType>();
```

Comment:
This statement might be redundant since `_GSOpen()` seems to handle the undefined case anyway. Leaving it in might be better for readability though.